rpm/SPECS/openresty-openssl.spec

```diff
@@ -66,19 +77,15 @@ rm -rf %{buildroot}
 
 %files
 %defattr(-,root,root,-)
-
-%attr(0755,root,root) %{openssl_prefix}/lib/*.so*
-%attr(0755,root,root) %{openssl_prefix}/lib/*/*.so*
-
+%attr(0755,root,root) %{openssl_prefix}/bin/openssl
+%attr(0755,root,root) %{openssl_prefix}/%{_lib}/*.so*
+%attr(0755,root,root) %{openssl_prefix}/%{_lib}/*/*.so*
```

Comment:
@agentzh I will try again to use lib, problem is that openssl does magic things to try and decide if files go in lib or lib64.